### ISSUE TITLE
Grasp: Integrate Spatial Softmax

### ISSUE THREAD

**Issue by ahundt** [jhu-lcsr/costar_plan#390]
This is lower priority but may be useful in future simulation training.

There is some work on this in both tensorflow and keras-contrib:
tensorflow/tensorflow#12549
https://github.com/keras-team/keras-contrib/pull/140
